bundles/org.openhab.binding.shelly/README.md

```diff
@@ -680,7 +792,22 @@ You can define 2 items (1 Switch, 1 Number) mapping to the same channel, see exa
 |battery   |batteryLevel |Number   |yes      |Battery Level in %                                                     |
 |          |lowBattery   |Switch   |yes      |Low battery alert (< 20%)                                              |
 
-### Shelly Button 1 (thing type: shellybutton1)
+### Shelly Motion (thing-type: shellymotion)
+
+|Group     |Channel        |Type     |read-only|Description                                                          |
+|----------|---------------|---------|---------|---------------------------------------------------------------------|
+|sensors   |motion         |Switch   |yes      |ON: Motion was detected                                              |
+|          |motionActive   |Switch   |yes      |ON: Motion is still active                                           |
```

Comment:
Why are there two Channels for this function? If you would use `Contact` instead of `Switch`, the user could easily select the motion icon for visualizing the state in the sitemap.

Reply:
removed, the device indicates both, one for the event, one showing there is still motion. Nevertheless they are changing the implementation so for now I removed the 2nd channel